src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/IUpgradeContextProperties.cs

```diff
@@ -1,12 +1,7 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System;
-using System.Collections;
 using System.Collections.Generic;
```

Comment:
I wish an analyzer would flag these so we don't have to do it manually periodically....

Reply:
I agree!